DigitalLibraryofTennessee/dltn_ingestion3_config_generator
# Add new sets for June ingest.

**GitHub Issue**: [June Ingest #227](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/227)

## What does this Pull Request do?

Adds newly added sets from UTK and UTC for the June DPLA ingest to our config file and sets list to share with Scott.

## What's new?

- utk_burford
- utk_hesler
- utk_jsevier
- utk_menbball
- utk_revwar
- utk_ramseyfp
- utk_tdh
- utk_womenbball
- utk_wwiioh
- utc_p16877coll34
- utc_p16877coll35
- utc_p16877coll36
- utc_p16877coll37
- utc_p16877coll38

## How should this be tested?

A description of what steps someone could take to:

Briefly check to make sure the records match those listed above.

## Additional Notes:

Ensure that we've resolved whatever is going on with the jsevier and ramseyfp sets before merging. Currently jsevier is returning 0 records.

## Interested parties

@markpbaggett
